lib/filters/mongodb_topology_filter.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+// console.log(argv._);
+var argv = require('optimist')
+  .usage('Usage: $0 -f [file] -e [environment]')
+  .demand(['f'])
+  .argv;
+
+var MongoDBTopologyFilter = function() {
+  this.filter = function(test) {
```

Comment:
maybe `this.filter = function(test, options) {`, where `options` can be some context that we build from the top layer that has _anything_ we want filters/plugins to use contextually. In this case we might add `argv` which is already parsed in the top level runner into the options passed to all plugins